evervault/http/request.py

```diff
@@ -50,12 +52,15 @@ def make_request(self, method, url, params=None, optional_headers={}, _is_ca=Fal
 
     def __build_headers(self, method, params, optional_headers, version):
         req_params = {}
+        auth_value = f"{self.app_uuid}:{self.api_key}"
+        encoded_auth_value_bytes = base64.b64encode(auth_value.encode("ascii"))
+        basic_auth_str = f"Basic {encoded_auth_value_bytes.decode('utf-8')}"
         headers = {
             "User-Agent": "evervault-python/" + version,
             "Accept-Encoding": "gzip, deflate",
             "Accept": "application/json",
             "Content-Type": "application/json",
-            "Api-Key": self.api_key,
```

Comment:
Isn't this going to break the SDK's ability to get relay's outbound destinations?